### PR TITLE
Add a section describing how the WT overview requirements are satisfied

### PR DESCRIPTION
**Pull request by afrind**
I tried stacking this PR on the previous commit (which also has a PR), but I'm not sure I did it correctly

### CODE REVIEW

**Comment by afrind** [comment]
Lol ok I totally didn't even read the issue, this definitely doesn't fix #4.

File: draft-kinnear-webtransport-http2.md [hunk 1]
```diff
@@ -48,6 +48,19 @@ author:
...
+    date: {DATE}
+    seriesinfo:
+      Internet-Draft: draft-ietf-webtrans-overview-latest
+    author:
+      -
+        ins: V. Vasiliev
+        name: Victor Vasiliev
+        organization: Google
+
+
```
Comment (ekinnear) [inline draft-kinnear-webtransport-http2.md:63]: Do we need this section still or is it getting picked up automatically by the latest version of the template?

File: draft-kinnear-webtransport-http2.md [hunk 2]
```diff
@@ -168,6 +181,29 @@ stream to the destination endpoint. The only frames allowed on WebTransport
...
+## Bidirectional Streams
+
+Once a WebTransport session is established, either endpoint can initiate a
+bidirectional stream by sending a WTHEADERS frame.
+
+## Datagrams
+
+TBD
+
```
Comment (ekinnear) [inline draft-kinnear-webtransport-http2.md:204]: This is a good start, I like the mini-section for each feature describing how to accomplish it with h2transport!

File: draft-kinnear-webtransport-http2.md (hunk 1 above)
Comment (afrind) [reply draft-kinnear-webtransport-http2.md:63]: I was getting build errors without it when I tried to reference [OVERVIEW]?

**Comment by ekinnear** [comment]
(Feel free to reopen if still relevant after H3-aligned rewrite, this was closed when transitioning default branch names)

**Pull request closed without merging**